Dockerfile.hub

```diff
@@ -11,12 +11,13 @@ LABEL org.opencontainers.image.source="https://github.com/Software-Improvement-G
 LABEL org.opencontainers.image.documentation="https://github.com/Software-Improvement-Group/sigridci/tree/main/docs"
 LABEL org.opencontainers.image.revision="${REVISION}"
 
-RUN apk --no-cache add git && adduser -S sigridci
+RUN set -eux; \
+	apt-get update; \
+	apt-get install -y --no-install-recommends git; \
+	rm -rf /var/lib/apt/lists/*
 
 COPY --link sigridci /sigridci
 
 ENV PATH /sigridci:$PATH
 
-USER sigridci
```

Comment:
This would mean we run as root, right?

Reply:
Yes. I'm asking around internally to see if there's any other way, but AFAIK the fifth requirement leaves no other way.